jetson/percep/main.cpp

```diff
@@ -72,9 +76,17 @@ int main() {
   cam.record_ar_init();
   #endif
 
-
+  Display display("Console Display");
 /* --- Main Processing Stuff --- */
   while (true) {
+    #if PERCEPTION_DEBUG
+      auto fpsStart = std::chrono::high_resolution_clock::now();
```

Comment:
Ideally we don't want to have a bunch of high resolution clock calls throughout our code, since this can make things quite messy. Ashwin has written a [timer class](https://github.com/umrover/perception-gpu-obs-detector/blob/main/include/Timer.hpp) that might be helpful here to integrate.